miner/block_builder.go

```diff
@@ -402,6 +402,7 @@ func (t *BlockBuilder) acceptBlockData() {
 		case layerID := <-t.beginRoundEvent:
 			if !t.syncer.IsSynced() {
 				t.Debug("builder got layer %v not synced yet", layerID)
+				continue
```

Comment:
what does this has to do with metrics ?

Reply:
Shaatnez shel Barak

Reply:
I'd still rather you put it in a different pr